### PR TITLE
Feature/ps 1131 pub and sync resync

### PR DESCRIPTION
**Pull request by Ruslan05**
- Developer(s): @Ruslan05

- Peer Reviewer:

- Ticket: https://spryker.atlassian.net/browse/PS-1131

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department
- [ ] All new or heavily changed classes get an "A" rating (Scrutinizer), except Facades, Factories, QueryContainers and Tests

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   EventBehavior         | minor        |                       |

#### Release Notes

Added support of console event:trigger command, which will refresh all the *_storage, *_search tables and run update for Redis and Elasticsearch storages. 

-----------------------------------------

#### Module EventBehavior

_**Minor:** Added functionality in a backwards-compatible manner_

Def of done (by responsible developer):
- [ ] All changes are backward-compatible. Outdated code is marked as deprecated.
- [ ] New and changed facade methods covered by functional tests. / Has nothing to test.
- [ ] New and changed complex business logic is covered by unit or integration tests. / Has nothing to test.

##### Change log

Initial Release

- Added cli command event:trigger
- New facade method `executeResolvedPluginsBySources` which accept resource names and call plugins to re-sync published data.

### CODE REVIEW

File: src/Spryker/Zed/EventBehavior/Business/Model/EventResourceManager.php [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+    /**
+     * @param \Spryker\Zed\EventBehavior\Dependency\Plugin\EventResourcePluginInterface $plugin
+     * @param array $ids
+     *
+     * @return void
+     */
+    protected function trigger(EventResourcePluginInterface $plugin, array $ids): void
+    {
+        //TODO replace this with triggerBulk as soon this method is available in Event module
```
Comment (ehsanmx) [inline src/Spryker/Zed/EventBehavior/Business/Model/EventResourceManager.php]: At least remove this todo for now and create a subticket in DataImporterOptimization ticket, we will fix this there

File: src/Spryker/Zed/EventBehavior/Business/Model/EventResourceRepositoryManager.php [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+    /**
+     * @param \Spryker\Zed\EventBehavior\Dependency\Plugin\EventResourcePluginInterface $plugin
+     * @param array $ids
+     *
+     * @return void
+     */
+    protected function trigger(EventResourcePluginInterface $plugin, array $ids): void
+    {
+        //TODO replace this with triggerBulk as soon this method is available in Event module
```
Comment (ehsanmx) [inline src/Spryker/Zed/EventBehavior/Business/Model/EventResourceRepositoryManager.php]: same here

**Comment by ehsanmx** [comment]
Please update template here!

File: src/Spryker/Zed/EventBehavior/Business/Model/EventResourceRepositoryManager.php (hunk 2 above)
Comment (Ruslan05) [reply src/Spryker/Zed/EventBehavior/Business/Model/EventResourceRepositoryManager.php]: Created sub-task: https://spryker.atlassian.net/browse/PS-2866

File: src/Spryker/Zed/EventBehavior/Business/Model/EventResourceManager.php (hunk 1 above)
Comment (Ruslan05) [reply src/Spryker/Zed/EventBehavior/Business/Model/EventResourceManager.php]: Created sub-task: https://spryker.atlassian.net/browse/PS-2866

**Pull request merged**